Yoast/javascript
# Clean up Dutch morphology

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Deletes an unnecessary file that was used for generating participle forms, and changes path names in line with changed naming in the morphology data file

## Test instructions

This PR can be tested by following these steps:

* enter some stems from the noVowelDoubling list (in morphology data file) followed by suffixes that start with -e (-e, -er, -en...) to determinStemSpec.js and makes sure the stem does not get the vowel doubled. For example: _bedragen_ should be stemmed to _bedrag_, not _bedraag_.


## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-239
